cap/modules/schemas/models.py

```diff
@@ -164,26 +164,42 @@ def record_path(self):
     @property
     def deposit_index(self):
         """Get deposit index name."""
-        path = urljoin(current_app.config['SCHEMAS_DEPOSIT_PREFIX'], str(self))
+        path = urljoin(
+            current_app.config.get('SEARCH_INDEX_PREFIX', '')
+            + current_app.config['SCHEMAS_DEPOSIT_PREFIX'],
+            str(self),
+        )
         return name_to_es_name(path)
 
     @property
     def record_index(self):
         """Get record index name."""
-        path = urljoin(current_app.config['SCHEMAS_RECORD_PREFIX'], str(self))
+        path = urljoin(
```

Comment:
do we need something else except from these changes in `deposit_index` `record_index`